locksmith/src/utils/getEventOrganizers.ts

```diff
@@ -0,0 +1,38 @@
+import { PaywallLocksConfigType } from '@unlock-protocol/core'
```

Comment:
I think the name of the file should be just `eventOrganizers` or `isEventOrganizer` if it will only contain that function.